File: app/src/main/java/com/example/meeera/browserapp/Model/ArticleDetail.java
package com.example.meeera.browserapp.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

/**
 * Created by meeera on 30/10/17.
 */

public class ArticleDetail extends RealmObject {
    @SerializedName("author")
    @Expose
    public String author;
    @SerializedName("title")
    @Expose
    public String title;
    @SerializedName("description")
    @Expose
    public String desc;
    @SerializedName("url")
    @Expose
    public String url;
    @SerializedName("urlToImage")
    @Expose
    public String urlToImage;
    @SerializedName("publishedAt")
    @Expose
    public String publishedAt;

    public String getAuthor() {
        return author;
    }

    public String getTitle() {
        return title;
    }

    public String getDesc() {
        return desc;
    }

    public String getUrl() {
        return url;
    }

    public String getUrlToImage() {
        return urlToImage;
    }

    public String getPublishedAt() {
        return publishedAt;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public void setUrlToImage(String urlToImage) {
        this.urlToImage = urlToImage;
    }

    public void setPublishedAt(String publishedAt) {
        this.publishedAt = publishedAt;
    }
}

File: app/src/main/java/com/example/meeera/browserapp/Model/ArticleDetail.kt
package com.example.meeera.browserapp.Model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by meeera on 13/10/17.
 */
class ArticleDetail {
    @SerializedName("author")
    @Expose
    val author : String = ""
    @SerializedName("title")
    @Expose
    val title : String = ""
    @SerializedName("description")
    @Expose
    val desc : String = ""
    @SerializedName("url")
    val url : String = ""
    @SerializedName("urlToImage")
    @Expose
    val urlToImage : String = ""
    @SerializedName("publishedAt")
    @Expose
    val publishedAt : String = ""
}